pyanaconda/installclasses/rhel.py

```diff
@@ -104,3 +103,52 @@ def setDefaultPartitioning(self, storage):
                     autoreq.fstype = storage.defaultFSType
 
         storage.autoPartitionRequests = autorequests
+
+    def filterSupportedLangs(self, ksdata, langs):
+        self._initialize_localemap(ksdata.ostreesetup.ref)
+        for lang in langs:
+            if lang in self.localemap:
+                yield lang
+
+    def filterSupportedLocales(self, ksdata, lang, locales):
+        self._initialize_localemap(ksdata.ostreesetup.ref)
+        supported = []
+        if lang in self.localemap:
+            for locale in locales:
+                stripped = self._strip_codeset_and_modifier(locale)
+                if stripped in self.localemap[lang]:
+                    supported.append(locale)
+        return supported
+
+    def _initialize_localemap(self, ref):
+
+        if self.localemap:
+            return
+
+        iutil.mkdirChain("/install/ostree/tmp/usr/lib")
+        rc = iutil.execWithRedirect("/usr/bin/ostree",
+            ["checkout", "--repo", "/install/ostree", ref,
```

Comment:
Well, I wonder if we should get the /install/ostree location from --url of ostreesetup command (supplied currently by interactive-defaults.ks via installer iso compose process), ie ksdata.ostreesetup.url so it survives possible changes in iso building process, or applies to user-supplied repo.